lnd_test.go

```diff
@@ -2724,10 +2724,11 @@ func testChannelForceClosure(net *lntest.NetworkHarness, t *harnessTest) {
 			return false
 		}
 
-		forceClose, predErr := findForceClosedChannel(
+		forceClose, err := findForceClosedChannel(
```

Comment:
Isn't it time to introduce a `WaitNoError`? It keeps trying the wait func in a loop until it returns no error. If the timer expires, it returns the last error, similar to what is happening here with `predErr`.

Reply:
Indeed, that would help. Having an api that returns the errors directly would be very nice

Reply:
It is small change

Reply:
Feel free to add it :) I would prefer it being added as we go rather than a massive change in `lnd_test` though.

Reply:
I don't mean a massive change. Just add the method and use it only here. Make a start with 'added as we go' .

Reply:
i'd support adding it here

Reply:
Added! PTAL @cfromknecht @joostjager